### ISSUE TITLE
Bioconductor dependencies fail

### ISSUE THREAD

**Issue by andrewrech** [r-lib/devtools#1649]
Hello,

Looks like the Bioconductor version bump [broke pulling remotes](https://github.com/hadley/devtools/blob/ad6f28ef9de6a02e1ea300af45d34deccc40bd2f/R/install-bioc.r#L57).

When I install my package with remote:

```
bioc::release/Biostrings
```

I get:
```
svn: E170013: Unable to connect to a repository at URL 'https://hedgehog.fhcrc.org/bioconductor/branches/RELEASE_3_6/madman/Rpacks/BiocInstaller'
svn: E175013: Access to '/bioconductor/branches/RELEASE_3_6/madman/Rpacks/BiocInstaller' forbidden
```

Not sure if this devtools or bioconductor issue, I am sorry if the latter.

Thank you very much

```
R version 3.4.2 (2017-09-28)
Platform: x86_64-apple-darwin17.0.0 (64-bit)
Running under: macOS High Sierra 10.13.1
```

**Comment by jimhester** [comment]
The devtools code needs to be updated to use git based Bioconductor.

**Comment by andrewrech** [comment]
Am I correct that all devtools Bioconductor installs are currently broken because of this?

**Comment by jimhester** [comment]
Not everything, only `install_bioc()` and the `bioc()` remote is broken. using `install()` locally would still work, as would `install_git()` with the Bioconductor repository, although that would always install the devel branch.

**Comment by andrewrech** [comment]
Ah, I did not realize this about `install_git`, thank you!

**Comment by rcannood** [comment]
I've attempted to fix this over at [dynverse/devtools](https://github.com/dynverse/devtools) by looking at the code for install_git. I've implemented the fix on the master branch, and backported it to 1.13.0.

This fix allows travis to install bioconductor packages from the git repostories.

Should I create a pull request to be reviewed?

**Comment by jimhester** [comment]
Yes please do!

**Comment by rcannood** [comment]
@jimhester Could you review my latest PR (#1705).

I've updated my repository as there were some merge conflicts in the meantime. 

Since the subversion repositories have, in the meantime, been fully discontinued, I think this issue should get looked at rather sooner than later.

**Comment by jimhester** [comment]
Closed by #1705
